cmd/multiFaToVcf/multiFaToVcf.go

```diff
@@ -20,14 +20,20 @@ func multiFaToVcf(inFile string, chr string, outFile string, substitutionsOnly b
 	out := fileio.EasyCreate(outFile)
 	header := vcf.NewHeader()
 	vcf.NewWriteHeader(out, header)
-	convert.PairwiseFaToVcf(f, chr, out, substitutionsOnly, retainN)
+	if len(f) == 2 {
+		convert.PairwiseFaToVcf(f, chr, out, substitutionsOnly, retainN)
+	} else if len(f) == 3 {
+		convert.ThreeWayFaToVcf(f, chr, out, substitutionsOnly, retainN)
+	}
 	err := out.Close()
 	exception.PanicOnErr(err)
 }
 
 func usage() {
 	fmt.Print(
-		"multiFaToVcf - Generates a VCF file from an input pairwise multiFa alignment with the first entry as the reference." +
```

Comment:
Let's at \n to the end of each of these lines. This will clean up the print statement